packages/ffe-file-upload-react/src/FileUpload.js

```diff
@@ -29,97 +33,103 @@ class FileUpload extends React.Component {
-            ),
-        );
+        this.props.onFileDeleted(this.props.files[event.target.id]);
     }
 
     render() {
         const {
             id,
             label,
-            selectedFilesHeaderLabel,
-            selectedFiles,
-            accept,
+            files,
+            cancelText,
+            deleteText,
             multiple,
-            errorMessage,
-            infoMessage,
-            successMessage,
+            title,
+            infoText,
+            infoSubText,
+            uploadTitle,
+            uploadMicroText,
+            uploadSubText,
+            onFileDeleted,
         } = this.props;
 
         return (
             <div className="ffe-file-upload">
-                <button
-                    className="ffe-file-upload__button"
-                    aria-invalid={String(!!errorMessage)}
-                    onClick={this.triggerUploadFileNativeHandler}
+                <div className="ffe-file-upload__title">{title}</div>
+                {Object.keys(files).length > 0 ? (
+                    <div className="ffe-file-upload__file-items-section">
+                        {Object.keys(files).map((file, index) => (
+                            <FileItem
+                                key={index}
+                                index={index}
+                                file={files[file]}
+                                cancelText={cancelText}
+                                deleteText={deleteText}
+                                onFileDeleted={onFileDeleted}
+                            />
+                        ))}
+                    </div>
+                ) : (
+                    <div className="ffe-file-upload__info-section">
+                        <div className="ffe-file-upload__info-section-text">
+                            {infoText}
+                        </div>
+                        <div className="ffe-file-upload__info-section-subtext">
+                            {infoSubText}
+                        </div>
+                    </div>
+                )}
+                <div
+                    className="ffe-file-upload__upload-section"
+                    role="presentation"
+                    onDrop={this.onFilesDropped}
+                    onDragOver={event => {
+                        event.preventDefault();
+                        this.setState({ hover: true });
+                    }}
+                    onDragLeave={() => this.setState({ hover: false })}
                 >
-                    <span className="ffe-file-upload__button__label-icon" />
-                    <span id={`${id}-label`}>{label}</span>
-                </button>
+                    <div
+                        className={`ffe-file-upload__upload-section-border${
+                            this.state.hover
+                                ? ' ffe-file-upload__upload-section-border__hover'
+                                : ''
+                        }`}
```

Comment:
You could rather use the `classnames` package here.

```
className={ classnames(
  'ffe-file-upload__upload-section-border',
  { 'ffe-file-upload__upload-section-border__hover': this.state.hover }
) }
```

The package is used alot throughout Designsystem.